uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/absence-tab/components/annual-entitlements.js

```diff
@@ -0,0 +1,163 @@
+/* eslint-env amd */
+
+define([
+  'common/lodash',
+  'common/moment',
+  'leave-absences/absence-tab/modules/components',
+  'common/models/contact'
+], function (_, moment, components) {
+  components.component('annualEntitlements', {
+    bindings: {
+      contactId: '<'
+    },
+    templateUrl: ['settings', function (settings) {
+      return settings.pathTpl + 'components/annual-entitlements.html';
+    }],
+    controllerAs: 'entitlements',
+    controller: ['$log', '$q', 'AbsenceType', 'AbsencePeriod', 'Entitlement', 'Contact', controller]
+  });
+
+  function controller ($log, $q, AbsenceType, AbsencePeriod, Entitlement, Contact) {
+    $log.debug('Component: annual-entitlements');
+
+    var vm = {};
+
+    vm.absenceTypes = [];
+    vm.absencePeriods = [];
+    vm.contactId = this.contactId;
+    vm.contacts = [];
+    vm.entitlements = [];
+    vm.loaded = { absencePeriods: false };
```

Comment:
For the existing components, the variables name is `vm.loading`, which is true by default, and changes to false when everything is loaded. Lets keep consistency, same for `contractEntitlements`.